CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+# [4.21.2](https://github.com/GetStream/stream-chat-swift/releases/tag/4.21.2)
+_September 19, 2022_
+
+## StreamChatUI
+### 🐞 Fixed
+- Fix the Message List cell not updating when a Reaction of the same type was updated but from the current user [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)
+- Fix the Message List cell not updating when the number of reactions of the same type changed [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)
```

Comment:
```suggestion
- Fix Message List cell not updating when the number of reactions of the same type changed [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)
```

Reply:
Also, I would not add 2 entries but just one. Maybe this last one

Reply:
This is actually 2 fixes, not 1, so it makes sense to have both. There's 1 issue related to reaction counts not increasing, and another one, which is current user reactions not appearing as well